samba.yaml

```
@@ -0,0 +1,417 @@
+package:
+  name: samba
+  version: 4.19.4
```

```suggestion
  version: 4.19.5
```

I see upstream [cut a new tag](https://gitlab.com/samba-team/samba/-/tags/samba-4.19.5), and it reduces the CVE count quite a bit.